config/application.rb

```diff
@@ -56,5 +57,7 @@ class Application < Rails::Application
     config.autoload_paths << Rails.root.join('lib')
 
     config.active_job.queue_adapter = :sidekiq
+
+    config.middleware.insert_before 0, SameSiteSecurity::Middleware
```

Comment:
Sorry, I am not sure I understand.
If I define `class SameSiteSecurityMiddleware ... end` in `lib/same_site_security_middleware.rb` 
and then I use 
```
config.middleware.insert_before 0, SameSiteSecurityMiddleware
```
I get `NameError: uninitialized constant Signonotron2::Application::SameSiteSecurityMiddleware`